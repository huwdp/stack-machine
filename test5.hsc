push 0
label main
print
push 1
add
push 10
jg main
