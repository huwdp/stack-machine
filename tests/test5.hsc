push 0
label main
printLine
push 1
add
push 10
jl main
