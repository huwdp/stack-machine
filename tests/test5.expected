Welcome to Huw's Stack Machine
0
1
2
3
4
5
6
7
8
9
